env.example

```diff
@@ -0,0 +1,10 @@
+export APP_DIR=/usr/app
+export APP_ENV=$RAILS_ENV
+export AWS_ACCESS_KEY_ID=c7b3rizH@rd
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_SECRET_ACCESS_KEY=c7b3rizH@rd
+export CONTAINER_HOST=352362988575.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
I'm also looking into the possibility of making this a dns solution ie: xo_docker_host.xogrp.com or something of that nature rather than the ugly ecr generated url